app/assets/javascripts/discourse/app/controllers/edit-topic-timer.js

```diff
@@ -121,9 +137,14 @@ export default Controller.extend(ModalFunctionality, {
 
   actions: {
     onChangeStatusType(value) {
-      if (value !== CLOSE_STATUS_TYPE) {
+      if (![CLOSE_AFTER_LAST_POST_STATUS_TYPE].includes(value)) {
```

Comment:
It's a bit odd to use `includes` for one value. I'd use `value !== CLOSE_AFTER_LAST_POST_STATUS_TYPE`

Reply:
Ah I did have 2 values there before and I forgot to remove the includes when I removed the other value!